src/main/java/se/sundsvall/webmessagecollector/integration/opene/OpenEClient.java

```diff
@@ -17,50 +17,52 @@
 import se.sundsvall.webmessagecollector.integration.opene.exception.OpenEException;
 
 @Component
-class OpenEClient {
-    private final OpenEIntegrationProperties properties;
-    
-    private final CloseableHttpClient httpClient;
-    
-    OpenEClient(OpenEIntegrationProperties properties) {
-        this.properties = properties;
-        this.httpClient = HttpClients.custom()
-            .setDefaultCredentialsProvider(getCredentials())
-            .build();
-    }
-    
-    byte[] getMessages(String familyId, String fromDate, String toDate) throws IOException {
-        return getBytes(buildUrl(properties.getMessagesPath(), Map.of("familyid", familyId), fromDate, toDate));
-    }
+public class OpenEClient {
 
-    private byte[] getBytes(URI url) throws IOException, OpenEException {
-        return httpClient.execute(new HttpGet(url), responseHandler -> {
-            if (responseHandler.getCode() == 200) {
-                return EntityUtils.toByteArray(responseHandler.getEntity());
-            }
+	private final OpenEIntegrationProperties properties;
+
+	private final CloseableHttpClient httpClient;
+
+	OpenEClient(final OpenEIntegrationProperties properties) {
+		this.properties = properties;
+		this.httpClient = HttpClients.custom()
+			.setDefaultCredentialsProvider(getCredentials())
+			.build();
+	}
+
+	byte[] getMessages(final String familyId, final String fromDate, final String toDate) throws IOException {
+		return getBytes(buildUrl(properties.getMessagesPath(), Map.of("familyid", familyId), fromDate, toDate));
+	}
+
+	private byte[] getBytes(final URI url) throws IOException, OpenEException {
+		return httpClient.execute(new HttpGet(url), responseHandler -> {
```

Comment:
Feign?

Reply:
I'm in the process of switching to Feign in another task